src/parser/dep.fs

```diff
@@ -330,8 +330,9 @@ let collect_one (verify_flags: list<(string * ref<bool>)>) (verify_mode: verify_
     | Pragma _ ->
         ()
     | TopLevelModule lid ->
-        raise (Err (Util.format1 "Automatic dependency analysis demands one \
-          module per file (module %s not supported)" (string_of_lid lid true)))
+        ()
+//        raise (Err (Util.format1 "Automatic dependency analysis demands one \
+//          module per file (module %s not supported)" (string_of_lid lid true)))
```

Comment:
Sure. Then it means that my error message about finding a `TopLevelModule` was wrong; it didn't mean that there were multiple modules in a file, but rather that this case just shouldn't happen.

If you can confirm that your proposed fix does a hard-fail in the case we discussed above, then I think we're good! Thanks!